docs/configuration.asciidoc

```diff
@@ -21,8 +21,9 @@ secret_token: <%= ENV["VERY_SECRET_TOKEN"] %>
 
 Options are applied in the following order (last one wins):
 
+1. Arguments to `ElasticAPM::<framework>.start`
```

Comment:
I don't like this. Can't we keep it like it is? It wasn't _not_ true.

Reply:
You mean don't change this list at all to include the `ElasticAPM::framework.start` references?

Reply:
Yeah, I think so. Or change it to just `.start` without the specific module/class.

Reply:
So this?

1. Defaults
2. Arguments to `ElasticAPM.start` / `Config.new`
3. Config file eg. `config/elastic_apm.yml`
4. Environment variables
5. Arguments to a framwork's `.start`. Ex: `ElasticAPM::Grape.start`

Reply:
But that's not true, is it?

What I meant:
1. Defaults
1. Arguments to `.start(...)`
1. Config file
1. ENV vars

Reply:
Yes, you're right. I keep confusing myself and forgetting I used `||=` in the `configure_app` method of the framework module. I've reverted my changes so it's back to the way it was.